src/features/Lock/lock.js

```diff
@@ -402,7 +402,9 @@ const Lock = () => {
                       wallet={wallet}
                       updateParentLockValue={lv => handleSetLockValue(lv)}
                       radioValue={radioValue}
-                      filledAmount={Number(filledAmount).toFixed(2)}
+                      filledAmount={
+                        filledAmount ? Number(filledAmount).toFixed(2) : 0
```

Comment:
whats the reason of this ternary here?

Reply:
It was included to avoid an undefined usage on the child component, instead we are just using 0.